events/giveaways/giveawayReactionAdded.js

```diff
@@ -1,62 +1,43 @@
 const { MessageEmbed, MessageButton, MessageActionRow } = require("discord.js")
 module.exports = {
   async execute(giveaway, reactor, messageReaction) {
```

Comment:
Same point here about renaming `giveaway`. Also at some point you'll want to clear out all the commented-out stuff. I understand it's fine for now since you're aping off what was already there 🦍